documentation/tutorials/kaggle_beginner_example_classification.ipynb

```diff
@@ -0,0 +1,2105 @@
+{
```

Comment:
<p>Notebooks need to run start-to-finish. If you want a user to do some work with incomplete or un-runnable code, consider using a code block in a markdown cell.</p><p> </p><p>Some other options:</p><ul><li>Rewrite this to show the user each step</li><li>Provide a short list of ideas (e.g. "For further exercise, try creating a model to do ...")</li></ul><p> </p><p>Keep in mind that these notebooks are published to tensorflow.org in a read-only format, so it probably doesn't make sense to say "Try it yourself, ...  your code here ... etc" when there's no way a user can interact.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/decision-forests/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
Apologies for this, this one was due to my own suggestion basing things off of some colabs I had seen previously! 

I think I wasn't considering the fact, when supplying feedback to Vansh, that the notebooks would be published in a read-only format not similar to what is viewable on colab. Apologies for that confusion I caused myself!

Reply:
No problem at all - if you want to make an interactive notebook instead, that's totally fine. We'll just need to exclude it from publication on tensorflow.org.

I do recommend getting it into a format we can use on the site though, you'll get a lot more visibility there.